src/Illuminate/Encryption/CipherMethods.php

```diff
@@ -21,6 +39,11 @@ public function supported()
         }
     }
 
+    /**
+     * Set key.
+     *
+     * @param string $key
+     */
```

Comment:
missing return annotation

Reply:
Is mixed correct?

Reply:
Actually this method has no return value at all, because it is a setter and the interface is not fluent. I really don't know how to hint this in docbloc... Any suggestions? :)

Reply:
Should be `void` then :0

Reply:
Should be void then :)